libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -1464,6 +1464,32 @@ void AP_Arming::set_aux_auth_failed(uint8_t auth_id, const char* fail_msg)
     }
 }
 
+void AP_Arming::reset_all_aux_auths()
+{
+    WITH_SEMAPHORE(aux_auth_sem);
+
+    // shortcut if this is not needed
+    if (aux_auth_count == 0) {
+        return;
+    }
+
+    // clear all auxiliary authorisation ids
+    aux_auth_count = 0;
+    // clear any previous allocation errors
+    aux_auth_error = false;
+
+    // reset states for all auxiliary authorisation ids
+    for (uint8_t i = 0; i < aux_auth_count; i++) {
```

Comment:
You already set this to zero at 1477, so this won't do anything no?

Reply:
Wow, yeah, meant to make this `aux_auth_count_max` originally, good catch.